build.sbt

```diff
@@ -52,8 +52,7 @@ lazy val catsDependencies = List(
   )
 
 libraryDependencies ++= Seq(
-  "org.broadinstitute" %% "wdl4s-wdl" % wdl4sV,
-  "org.broadinstitute" %% "wdl4s-cwl" % wdl4sV,
+  "org.broadinstitute" %% "wdl4s" % wdl4sV,
```

Comment:
not sure I would do this, the existence of this artifact was purely an accident and may be corrected